### PR TITLE
Document --python flag

### PR DESCRIPTION
**Pull request by MaciejG604**

### CODE REVIEW

**Review by tgodzik — APPROVED** [review]
Looks good! We can mention it in Scala Space later

File: website/docs/guides/python.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  randoms.foreach(println(_))
+}
+
+val numbers = py"[x * 2 for x in ${Iterator.from(3).take(10).toList.toPythonCopy}]"
+  .as[Seq[Int]]
+
+println(numbers)
+```
+
+You can also use Scala Native to create a native binary with direct bindings to CPython.
```
Comment (tgodzik) [inline website/docs/guides/python.md:55]: Maybe let's add what libs are needed here and how one can install them simplest

File: website/docs/guides/python.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+      }
+    }
+    sequences
+  }
+}
+
+```
+Run:
+```bash
+scala-cli package --native PlotDemo.scala -o plot
```
Comment (Gedochao) [inline website/docs/guides/python.md]:
```suggestion
scala-cli package --native --power PlotDemo.scala -o plot
```

**Comment by Gedochao** [comment]
We should also add the guide in the guides index at `guides/intro.md`

**Review by Gedochao — APPROVED** [review]

**Pull request merged**